src/components/ProjectsCard/ProjectsCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "@components/Button/Button";
+import { Text } from "@components/Text/Text";
+import { IconNames } from "@data/interfaces/Icons";
+import styles from "./ProjectsCard.module.css";
+import { IProjectsCardProps } from "@data/interfaces/Card";
+const ProjectsCard = ({
+  projectTitle,
+  projectDescription,
+  projectUrl,
+  projectInfoUrl,
+  projectPictureUrl,
+  cardOrientation,
+}: IProjectsCardProps) => {
+  const isReversed = cardOrientation === "reversed";
+
+  return (
+    <article className={styles.projectsCard}>
```

Comment:
Use more relevant class names. This article is a single project, not the hole list
``` <article className={styles.projectsCard}> ```